index.js

```diff
@@ -59,6 +59,10 @@ module.exports = {
     // https://eslint.org/docs/rules/no-var
     'no-var': 2,
 
+    // require at least one whitespace after comments( // and /*)
+    // https://eslint.org/docs/rules/spaced-comment
+    'spaced-comment': ["error", "always"],
```

Comment:
In this file, we also follow our eslint rules, so simple quotes.

For that kind of rules, I usually start with a simple 'warn' to make sure we don't break CI in other projects.

Reply:
i see, makes sense. will update.